src/java.desktop/share/classes/sun/print/RasterPrinterJob.java

```diff
@@ -1654,11 +1647,6 @@ protected void validateDestination(String dest) throws PrinterException {
         } catch (IOException ioe) {
             throw new PrinterException("Cannot write to file:"+
                                        dest);
-        } catch (SecurityException se) {
-            //There is already file read/write access so at this point
```

Comment:
Few other comments seems obsolete now

line 948:

```
/* A security check has already been performed in the
 * java.awt.print.printerJob.getPrinterJob method.
 * So by the time we get here, it is OK for the current thread
 * to print either to a file (from a Dialog we control!) or
 * to a chosen printer.
 *
 * We raise privilege when we put up the dialog, to avoid
 * the "warning applet window" banner.
 */
```
(Same for the `PSPrinterJob.java:508`)


line 1809:
```
     * The caller must have security permission to read system properties.
```

Reply:
ok